core/src/main/scala/org/apache/spark/ui/ToolTips.scala

```diff
@@ -99,4 +99,16 @@ private[spark] object ToolTips {
        dynamic allocation is enabled. The number of granted executors may exceed the limit
        ephemerally when executors are being killed.
     """
+  
+  val THRIFT_SERVER_FINISH_TIME =
+    "Finish time of the execution, before fetching the results"
```

Comment:
I think "Execution finish time" would sound better than "Finish time of the execution"

Reply:
I'm Sorry, spanish common error when we write in english :-)